app/models/hall.rb

```diff
@@ -0,0 +1,7 @@
+class Hall < ApplicationRecord
+    has_many :seances, dependent: :destroy
+
+    validates :capacity, :number, presence: true 
+    validates :capacity, :number, numericality: {only_integer: true, greater_than: 0 }
+    validates :number, uniqueness: true
```

Comment:
are you sure it should be the number? why not name? what if we want to name halls: red, white, or kebab?

Reply:
Name really does fit better here, thanks!